### DISCUSSION TITLE
When is uSyncMigrations compatible with Umbraco 11?

### DISCUSSION THREAD

**Discussion by anna-Kop** [Jumoo/uSyncMigrations#68] [Q&A]
Hello!
Is there any estimate when uSyncMigrations will work with Umbraco 11?
Thank you :)

**Comment by Wolf1757** [comment]
Hello,

I would be interessed to use this Package  with the latest Umbraco Version (11) to migrate an old Website which runs on Umbraco 7.

Do you know when this packege will be finally available for Umbraco 11?

Thanks You

**Comment by KevinJump** [comment]
Hi, My sneaking suspicion is that it will work with v11 now, (with uSync v11 installed) - but i haven't tried it. 

I will do some tests today - as we are due to release a weekly update.

Kevin

**Comment by giuunit** [comment]
Hi. We are running a v11 project and experimenting with uSyncMigrations. We were able to upload zip files, select files, get the tool to render the validation results, add a custom migrator and run the process behind the 'convert files' button. We have some minor tweaks on our end, but I would say the tool works with v11, nothing major blocking people from using it with v11.

**Comment by Wolf1757** [comment]
Here is my feedback: it worked like a charm. I was able to successfully migrate 2 websites (Umbraco 7.3 --> 11)

Thank you for your job!
